app/src/controllers/administrate/actionCreators.js

```diff
@@ -14,8 +14,13 @@
  * limitations under the License.
  */
 
-import { FETCH_PROJECT_DATA } from './constants';
+import { FETCH_PROJECT_DATA, FETCH_ACTIVE_PROJECT_DATA } from './constants';
 
 export const fetchProjectDataAction = () => ({
   type: FETCH_PROJECT_DATA,
 });
+
+export const fetchActiveProjectDataAction = (payload) => ({
+  type: FETCH_ACTIVE_PROJECT_DATA,
+  payload,
+});
```

Comment:
The same comment as for [this approach](https://github.com/reportportal/service-ui/pull/3876#discussion_r1648854041).
Let's not use the deprecated 'administrate' controllers and pages as they will be commissioned soon in favor of instance/organization level controllers.
Please move all your new logic to the organization controller or project controller.

Reply:
This is a repetition of the same comment in different threads, it needs to be edited in one place.

Reply:
Done.